.github/workflows/test-lightwalletd.yml

```
@@ -0,0 +1,370 @@
+name: Test lightwalletd
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        default: 'Mainnet'
+        description: 'Network to deploy: Mainnet or Testnet'
+        required: true
+      regenerate-disks:
+        type: boolean
+        default: false
+        description: 'Just update stateful disks'
```

Question: Does this just update the lightwalletd cached state, or also the Zebra cached state?

```suggestion
        description: 'Just update lightwalletd stateful disks'
```